rhel/SPECS/pmm-update.spec

```diff
@@ -40,12 +54,16 @@ cp -pav ./bin/* %{buildroot}%{_bindir}
 install -d %{buildroot}%{_datadir}/%{name}
 cp -pav ./ansible %{buildroot}%{_datadir}/%{name}
 
+cd src/github.com/percona/pmm-update
+install -p -m 0755 bin/pmm2-update %{buildroot}%{_bindir}
+
 
 %files
 %license LICENSE
 %doc README.md
 %{_bindir}/*
 %{_datadir}/%{name}
+/usr/lib/debug/usr/bin/pmm2-update.debug
```

Comment:
%{_libdir}?

Reply:
We do not have full macro for /usr/lib/debug/usr/bin, and the usage of _libdir does not make any sense. It will be something like   %{_libdir}/debug/usr/bin/pmm2-update.debug